From FEDORA_PATCHES Mon Sep 17 00:00:00 2001
From: Nick Clifton <nickc@redhat.com>
Date: Mon, 28 Apr 2025 11:23:22 -0700
Subject: gdb-rhel-86801-binutils-z17-update-5of12.patch

;; Backkport "Fix building for the s390 target with clang"
;; (Nick Clifton, RHEL-86801)

diff --git a/opcodes/s390-opc.c b/opcodes/s390-opc.c
--- a/opcodes/s390-opc.c
+++ b/opcodes/s390-opc.c
@@ -262,9 +262,10 @@ const struct s390_operand s390_operands[] =
 
 };
 
-static inline void unused_s390_operands_static_asserts(void)
+static inline void ATTRIBUTE_UNUSED
+unused_s390_operands_static_asserts (void)
 {
-  static_assert(ARRAY_SIZE(s390_operands) - 1 == J32_16);
+  static_assert (ARRAY_SIZE (s390_operands) - 1 == J32_16);
 }
 
 /* Macros used to form opcodes.  */
